test/test_find_uvar.py

```diff
@@ -114,15 +120,16 @@ def test_two_triangles_without_faces():
     grid = two_triangles_with_depths()
     grid.faces = None
 
-    grid.save_as_netcdf('2_triangles_without_faces.nc')
-
-    # Read it back in and check it out.
-    ug = UGrid.from_ncfile('2_triangles_without_faces.nc', load_data=True)
+    fname = '2_triangles_without_faces.nc'
+    with chdir(test_files):
+        grid.save_as_netcdf(fname)
+        ug = UGrid.from_ncfile(fname, load_data=True)
+        os.remove(fname)
```

Comment:
maybe we should write all the generated files to one location, then have a break down fixture that cleans it all up.

One of the tricks with this is that if a test is failing, it's nice to have the generated files around to look at.

I haven't settled on a canonical way to handle this yet myself.